cmd/conformance/mysql/README.md

```diff
@@ -1,54 +1,70 @@
-# How to Run a Tessera Log (MySQL)
+# Conformance MySQL log
+
+This binary runs an HTTP web server that accepts POST HTTP requests to a `/add` endpoint.
+This endpoint takes arbitrary data and adds it to a MySQL based Tessera log.
 
 > [!WARNING]
 > - This is an example and is not fit for production use, but demonstrates a way of using the Tessera Log with MySQL storage backend.
 > - This example is built on the [tlog tiles API](https://c2sp.org/tlog-tiles) for read endpoints and exposes a /add endpoint that allows any POSTed data to be added to the log.
 
-The tessera log with the MySQL storage implementation can be started with either Docker Compose or manual `go run`.
+## Bring up a log
+
+This will help you bring up a MySQL database to store a Tessera log, and start a personality
+binary that can add entries to it.
+
+You can either start this log using Docker Compose, or manually with `go run`.
 
 Note that all the commands are executed at the root directory of this repository.
 
-## Docker Compose
+### Docker Compose
 
-### Prerequisites
+#### Prerequisites
 
-- [Docker Compose](https://docs.docker.com/compose/install/)
+Install [Docker Compose](https://docs.docker.com/compose/install/).
 
-### Starting
+#### Start the log
 
 ```sh
 docker compose -f ./cmd/conformance/mysql/docker/compose.yaml up
 ```
 
 Add `-d` if you want to run the log in detached mode.
 
-### Stopping
+#### Stop the log
 
 ```sh
 docker compose -f ./cmd/conformance/mysql/docker/compose.yaml down
 ```
 
-## Manual 
+### Manual 
+
+#### Prerequisites
 
-### Prerequisites
+You need to have a MySQL database running on port `3306`.
```

Comment:
Will the reader also need to have configured the MySQL instance to accept password auth for `root` and set the password to `root`, and created a DB instance called `test_tessera`?

Reply:
@roger2hk

Reply:
Yes, the "starting the log" command in the next few lines requires the user and password to be `root`. We can turn them into variables, but I think it's easier to define them for easy & paste.

Reply:
If we don't want to use the `root` user, we can define the `MYSQL_DATABASE`, `MYSQL_USER`, `MYSQL_PASSWORD` in the `docker run` command.

Reply:
Alright, added some instructions.